Winapp2.ini

```diff
@@ -8641,6 +8575,7 @@ FileKey2=%AppData%\iMobie\AnyTrans\Backup|*.*|RECURSE
 FileKey3=%AppData%\iMobie\AnyTrans\ErrorLog|*.*|RECURSE
 FileKey4=%AppData%\iMobie\AnyTrans\iMobieConfig|*.*|RECURSE
 FileKey5=%AppData%\iMobie\Backup|*.*|RECURSE
+FileKey6=%AppData%\iMobie\DownloadWebFile\ErrorLog|*.log
```

Comment:
Separate the backup files out of this entry (github wouldn't let me tag the right line).